### DISCUSSION TITLE
We found the model outputs like logits are very different when using eager vs flash attention on TPU with the same input, model, etc. except the attention choice

### DISCUSSION THREAD

**Discussion by weirayao** [AI-Hypercomputer/torchprime#386] [Q&A]
As title. Is this expected?
